editor.planx.uk/src/@planx/components/FindProperty/Public/FindProperty.stories.tsx

```diff
@@ -1,27 +1,65 @@
+import { MockedProvider } from "@apollo/client/testing";
+import Wrapper from "@planx/components/fixtures/Wrapper";
 import { Meta, StoryObj } from "@storybook/react";
 import React from "react";
 
+import Editor from "../Editor";
 import FindProperty from "./";
+import findAddressReturnMock from "./mocks/findAddressReturnMock";
+import localAuthorityMock from "./mocks/localAuthorityMock";
 
-const metadata: Meta = {
+/** FindProperty relies on a custom web component that cannot be shown by React Storybook. Find additional docs here: https://oslmap.netlify.app/ */
+export default {
   title: "PlanX Components/FindProperty",
-  argTypes: {
-    handleSubmit: { action: true, control: { disable: true } },
-  },
+  component: FindProperty,
   parameters: {
-    reactNavi: {
-      useCurrentRoute: () => {
+    swr: {
+      default: () => {
         return {
-          data: {
-            team: "canterbury",
-          },
+          data: localAuthorityMock,
```

Comment:
This `swr` mock is currently unused - I _think_ it's actually mocked correctly, but the address-autocomplete doesn't render in storybook because it's a custom web component so it's never actually called. Might be interesting to try and come back to these cases, but I wasn't quite ready to get into the react vs web component webpack settings in storybook config here just yet.

For components relying on web components, I've made a note in their descriptions & linked to the Pitsby docs for now